activiti-services/activiti-services-rest/src/main/java/org/activiti/services/rest/resources/BpmnModelResource.java

```diff
@@ -0,0 +1,15 @@
+package org.activiti.services.rest.resources;
+
+import org.activiti.services.core.model.MetaBpmnModel;
+import org.springframework.hateoas.Link;
+import org.springframework.hateoas.Resource;
+
+public class BpmnModelResource extends Resource<MetaBpmnModel> {
```

Comment:
@daisuke-yoshimoto I think that we don't need these resources. Unless you find a clear advantage of this approach I would recommend to not have this extra layer of complexity here.